tests/queries/0_stateless/00991_system_parts_race_condition_long.sh

```diff
@@ -74,7 +74,12 @@ clickhouse_client_loop_timeout $TIMEOUT thread3 2> /dev/null &
 clickhouse_client_loop_timeout $TIMEOUT thread4 2> /dev/null &
 clickhouse_client_loop_timeout $TIMEOUT thread5 2> /dev/null &
 
+# wait until the queries above will start,
+# so that clickhouse_test_wait_queries will see them.
+sleep 5
 
-wait
+clickhouse_test_wait_queries 300
```

Comment:
Why do we care about it? Consider just disabling `left_queries_check` in `clickhouse-test`. It's not clear why do we need this check at all.